WordPress/src/main/java/org/wordpress/android/ui/compose/components/SolidCircle.kt

```diff
@@ -0,0 +1,26 @@
+package org.wordpress.android.ui.compose.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.Dp
+
+@Composable
+fun SolidCircle(
```

Comment:
Thank you for adding this reusable component 👍

Reply:
> Thank you for adding this reusable component 👍
Thanks @antonis 🙇🏻 
For naming I took inspiration from SwiftUI which calls it `Circle`. Ajesh was also interested in this component to potentially replace a [different solution](https://github.com/wordpress-mobile/WordPress-Android/pull/17941/files#diff-b85e2edbb71dce7062cc0c64df4bd322419a4379c191fe00b66dd42ca9fed120R258-R263) added recently for Blaze 👍🏻